{"uid":"a2cd6f57d9cc9eaa","name":"test_page_title","fullName":"tests.test_example.ExampleTestCase#test_page_title","historyId":"1f00af4742c2562b8d24e8127e7405ec","time":{"start":1757402718194,"stop":1757402720477,"duration":2283},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_page_title>\n\n    def test_page_title(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fa0ecf35e90>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1757402714570,"stop":1757402714570,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1757402722788,"stop":1757402722788,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"pkrvm7jw40e0xgp"},{"name":"thread","value":"2663-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"767333547e55aaf3","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.80)\nStacktrace:\n#0 0x560b77d6d94a <unknown>\n#1 0x560b777e28a0 <unknown>\n#2 0x560b777d9ce8 <unknown>\n#3 0x560b777cb1b6 <unknown>\n#4 0x560b777ccf00 <unknown>\n#5 0x560b777cb55e <unknown>\n#6 0x560b777caee7 <unknown>\n#7 0x560b777cabc5 <unknown>\n#8 0x560b777c8a42 <unknown>\n#9 0x560b777c922a <unknown>\n#10 0x560b777e5db9 <unknown>\n#11 0x560b77880835 <unknown>\n#12 0x560b7785a032 <unknown>\n#13 0x560b7787fc23 <unknown>\n#14 0x560b77859e03 <unknown>\n#15 0x560b77826968 <unknown>\n#16 0x560b778275e1 <unknown>\n#17 0x560b77d31548 <unknown>\n#18 0x560b77d35272 <unknown>\n#19 0x560b77d18313 <unknown>\n#20 0x560b77d35dc5 <unknown>\n#21 0x560b77cfd49f <unknown>\n#22 0x560b77d5a158 <unknown>\n#23 0x560b77d5a332 <unknown>\n#24 0x560b77d6ca53 <unknown>\n#25 0x7f53cd89caa4 <unknown>\n#26 0x7f53cd929c3c <unknown>","time":{"start":1757402716968,"stop":1757402717582,"duration":614}},{"uid":"b6edc2d7a770eeef","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402706977,"stop":1757402709664,"duration":2687}},{"uid":"31b3238bfb1c77b0","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402705333,"stop":1757402707482,"duration":2149}},{"uid":"1026f827802132a7","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x560c9e6aa08a <unknown>\n#1 0x560c9e149a70 <unknown>\n#2 0x560c9e140cf1 <unknown>\n#3 0x560c9e132099 <unknown>\n#4 0x560c9e133de0 <unknown>\n#5 0x560c9e13243e <unknown>\n#6 0x560c9e131daa <unknown>\n#7 0x560c9e131a85 <unknown>\n#8 0x560c9e12f8ee <unknown>\n#9 0x560c9e1300ca <unknown>\n#10 0x560c9e14cf99 <unknown>\n#11 0x560c9e1e7d65 <unknown>\n#12 0x560c9e1c13e2 <unknown>\n#13 0x560c9e1e714f <unknown>\n#14 0x560c9e1c11b3 <unknown>\n#15 0x560c9e18d59b <unknown>\n#16 0x560c9e18e971 <unknown>\n#17 0x560c9e66f25b <unknown>\n#18 0x560c9e672fa9 <unknown>\n#19 0x560c9e656339 <unknown>\n#20 0x560c9e673b58 <unknown>\n#21 0x560c9e63ac1f <unknown>\n#22 0x560c9e697118 <unknown>\n#23 0x560c9e6972f6 <unknown>\n#24 0x560c9e6a9066 <unknown>\n#25 0x7fb8b5a9caa4 <unknown>\n#26 0x7fb8b5b29c3c <unknown>","time":{"start":1757402703063,"stop":1757402703740,"duration":677}},{"uid":"125a5022043cc88d","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x557fb0bff08a <unknown>\n#1 0x557fb069ea70 <unknown>\n#2 0x557fb0695cf1 <unknown>\n#3 0x557fb0687099 <unknown>\n#4 0x557fb0688de0 <unknown>\n#5 0x557fb068743e <unknown>\n#6 0x557fb0686daa <unknown>\n#7 0x557fb0686a85 <unknown>\n#8 0x557fb06848ee <unknown>\n#9 0x557fb06850ca <unknown>\n#10 0x557fb06a1f99 <unknown>\n#11 0x557fb073cd65 <unknown>\n#12 0x557fb07163e2 <unknown>\n#13 0x557fb073c14f <unknown>\n#14 0x557fb07161b3 <unknown>\n#15 0x557fb06e259b <unknown>\n#16 0x557fb06e3971 <unknown>\n#17 0x557fb0bc425b <unknown>\n#18 0x557fb0bc7fa9 <unknown>\n#19 0x557fb0bab339 <unknown>\n#20 0x557fb0bc8b58 <unknown>\n#21 0x557fb0b8fc1f <unknown>\n#22 0x557fb0bec118 <unknown>\n#23 0x557fb0bec2f6 <unknown>\n#24 0x557fb0bfe066 <unknown>\n#25 0x7f3654a9caa4 <unknown>\n#26 0x7f3654b29c3c <unknown>","time":{"start":1757402702498,"stop":1757402703180,"duration":682}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"a2cd6f57d9cc9eaa.json","parameterValues":[]}